packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -30,13 +29,18 @@ class AmplifyDataStore extends DataStorePluginInterface {
       : super(token: _token, modelProvider: modelProvider);
 
   static AmplifyDataStore _instance = AmplifyDataStoreMethodChannel();
-  static AmplifyDataStoreEventChannel events = AmplifyDataStoreEventChannel();
+  static DataStoreStreamController streamWrapper = DataStoreStreamController();
+  ModelProviderInterface models;
 
   static set instance(DataStorePluginInterface instance) {
     PlatformInterface.verifyToken(instance, _token);
     _instance = instance;
   }
 
+  StreamController get streamController {
```

Comment:
Hm. Do you need to expose access to this publicly?

Reply:
Yes - It's called by core (now amplify_flutter).